next.config.js

```diff
@@ -32,13 +37,30 @@ module.exports = (phase, { defaultConfig }) => {
         use: "@svgr/webpack",
       })
 
+      // Tree shake Sentry debug code
+      // ref. https://docs.sentry.io/platforms/javascript/guides/nextjs/configuration/tree-shaking/#tree-shaking-with-nextjs
+      config.plugins.push(
+        new webpack.DefinePlugin({
+          __SENTRY_DEBUG__: false,
+          __RRWEB_EXCLUDE_IFRAME__: true,
+          __RRWEB_EXCLUDE_SHADOW_DOM__: true,
+          __SENTRY_EXCLUDE_REPLAY_WORKER__: true,
+        })
+      )
+
       return config
     },
     i18n,
     trailingSlash: true,
     images: {
       deviceSizes: [640, 750, 828, 1080, 1200, 1504, 1920],
     },
+    env: {
+      NEXT_PUBLIC_CONTEXT: process.env.CONTEXT,
```

Comment:
Im exposing this env var to segment the reports better. https://docs.netlify.com/configure-builds/environment-variables/#build-metadata

> It can be production, deploy-preview, branch-deploy, or dev.

We could also just trace things on `production` contexts.